workshops/book-dash/before_workshop.md

```diff
@@ -0,0 +1,131 @@
+Please sign up for a GitHub account if you don't already have one: https://github.com/join
+If you're not familiar with contributing to GitHub projects, we recommend you read our chapter on [Collaborating on GitHub](https://the-turing-way.netlify.com/collaborating_github/collaborating_github.html).
+Our [contributing guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) are also a good thing to read, but we will go through the main points again on the book dash day.
+We hope you already have an idea of what you would like to contribute, but if you are still looking for inspiration, our [list of good first issues](https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) will hopefully provide some inspiration. 
+Also, a reminder that everyone who participates in the Turing Way project is expected to abide by our [Code of Conduct](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE_OF_CONDUCT.md) - please familiarise yourself with its contents but we will also emphasise the main points again on the book dash day.
+
+## Networking evening
+
+We're looking forward to dining with you in a private room at the [Megaro Hotel](https://www.themegaro.co.uk).
+The kitchen need to order in the ingredients to cook up a delicious meal, so please pick your selection of dishes by the end of the day on Monday 20 May (sorry for the short turn around) through [this Google form](https://forms.gle/NsU76koETkaHfsLd8).
+
+Agenda for the evening will be as follows:
+6:30pm	Arrive at dinner venue   
+7:00pm	Dinner  
+8:30pm	Lightning talks  
+9:00pm	Drinks and networking   
+10:30pm Go home (get some sleep)  
+
+### Lightning talks / elevator pitches
+We have scheduled some time for you to give lightning talks about projects you're working on or anything else you are excited about.
+Those talks do not need to be related to the Turing Way or anything you want to contribute during the book dash - they can be about data visualization, your favourite R package, eccentric keyboards, meditation and wellbeing tips, beer brewing or cross stitching.
+Talks should be 3 minutes long and be without the support of slides as the venue will not be able to provide any equipment to display them.
+Please let us know on arrival if you would like to give an elevator pitch.
+
+## On the day
+
+We are very excited that an illustrator from [scriberia](http://www.scriberia.co.uk/) will join us to document the sprint and provide some amazing visualisations for the handbook.
+
+### Getting to the venue
+
+We will start the book dash before the British Library officially opens at 9:30am so you will need to enter through Gate 8 of The British Library, the staff entrance on Midland Road opposite (one of the exits from) St Pancras station ([here are instructions on how to get to The Alan Turing Institute](https://www.turing.ac.uk/about-us/how-get-alan-turing-institute)). 
```

Comment:
Can you change this link to https://whitakerlab.github.io/visit? It has a nice picture of the gate which I think makes it easier for folks to follow.